app/api/helpers/static.py

```diff
@@ -0,0 +1,397 @@
+    ],
+    'Charity & Causes': [
+        'Animal Welfare', 'Disaster Relief', 'Education',
+        'Environment', 'Healthcare', 'Human Rights',
+        'International Aid', 'Other', 'Poverty'
+    ],
+    'Community & Culture': [
+        'City/Town', 'County', 'Heritage', 'LGBT', 'Language',
+        'Medieval', 'Nationality', 'Other', 'Renaissance', 'State'
+    ],
+    'Family & Education': [
+        'Alumni', 'Baby', 'Children &amp; Youth', 'Education', 'Other',
+        'Parenting', 'Parents Association', 'Reunion'
+    ],
+    'Fashion & Beauty': [
+        'Accessories', 'Beauty', 'Bridal', 'Fashion', 'Other'
+    ],
+    'Film, Media & Entertainment': [
+        'Adult', 'Anime', 'Comedy', 'Comics', 'Film', 'Gaming', 'Other', 'TV'
+    ],
+    'Food & Drink': ["Beer", "Food", "Other", "Spirits", "Wine"],
+    'Government & Politics': [
+        "County/Municipal Government", "Democratic Party", "Federal Government",
+        "Non-partisan", "Other", "Other Party", "Republican Party",
+        "State Government"
+    ],
+    'Health & Wellness': [
+        "Medical", "Mental health", "Other", "Personal health", "Spa", "Yoga"
+    ],
+    'Hobbies & Special Interest': [
+        "Adult", "Anime/Comics", "Books", "DIY", "Drawing & Painting", "Gaming",
+        "Knitting", "Other", "Photography"
+    ],
+    'Home & Lifestyle': ["Dating", "Home & Garden", "Other", "Pets & Animals"],
+    'Music': [
+        "Alternative", "Blues & Jazz", "Classical", "Country", "Cultural",
+        "EDM / Electronic", "Folk", "Hip Hop / Rap", "Indie", "Latin", "Metal",
+        "Opera", "Other", "Pop", "R&B", "Reggae", "Religious/Spiritual", "Rock",
+        "Top 40"
+    ],
+    'Other': ["Avatar", "Logo"],
+    'Performing & Visual Arts': [
+        "Ballet", "Comedy", "Craft", "Dance", "Fine Art", "Literary Arts",
+        "Musical", "Opera", "Orchestra", "Other", "Theatre"
+    ],
+    'Religion & Spirituality': [
+        "Buddhism", "Christianity", "Eastern Religion", "Islam", "Judaism",
+        "Mormonism", "Mysticism and Occult", "New Age", "Other", "Sikhism"
+    ],
+    'Science & Technology': [
+        "Biotech", "High Tech", "Medicine", "Mobile", "Other", "Robotics",
+        "Science", "Social Media"
+    ],
+    'Seasonal & Holiday': [
+        "Channukah", "Christmas", "Easter", "Fall events", "Halloween/Haunt",
+        "Independence Day", "New Years Eve", "Other", "St Patricks Day",
+        "Thanksgiving"
+    ],
+    'Sports & Fitness': [
+        "Baseball", "Basketball", "Cycling", "Exercise", "Fighting & Martial Arts",
+        "Football", "Golf", "Hockey", "Motorsports", "Mountain Biking",
+        "Obstacles", "Other", "Rugby", "Running", "Snow Sports", "Soccer",
+        "Swimming & Water Sports", "Tennis", "Volleyball", "Walking", "Yoga"
+    ],
+    'Travel & Outdoor': [
+        "Canoeing", "Climbing", "Hiking", "Kayaking", "Other", "Rafting", "Travel"
+    ]
+}
+PAYMENT_COUNTRIES = [
```

Comment:
Countries to be referred to by the ISO two letter country code only. 
https://github.com/fossasia/open-event-frontend/blob/development/app/utils/dictionary/demography.js